<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Goudy Old Style</family>
    <pointsize>16</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Physical simulation of motion</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resurces/res.qrc">
    <normaloff>:/Resurses_2/icon.png</normaloff>:/Resurses_2/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Resurses_2/ФонБезВотерки.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Goudy Old Style</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Resurses_2/кремовый.jpg);
border-top: 4px solid #0B1D9D;
border-left: 4px solid #0B1D9D;
border-bottom: 4px solid #0B1D9D;
border-radius: 6px;</string>
    </property>
    <property name="text">
     <string>U(0)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_out_speed_start">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Goudy Old Style</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Resurses_2/кремовый.jpg);
border-top: 4px solid #0B1D9D;
border-right: 4px solid #0B1D9D;
border-bottom: 4px solid #0B1D9D;
border-radius: 6px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_out_speed_fin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Goudy Old Style</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Resurses_2/кремовый.jpg);
border-top: 4px solid #0B1D9D;
border-right: 4px solid #0B1D9D;
border-bottom: 4px solid #0B1D9D;
border-radius: 6px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Goudy Old Style</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Resurses_2/кремовый.jpg);
border-top: 4px solid #0B1D9D;
border-left: 4px solid #0B1D9D;
border-bottom: 4px solid #0B1D9D;
border-radius: 6px;</string>
    </property>
    <property name="text">
     <string>U(к)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Goudy Old Style</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Resurses_2/кремовый.jpg);
border-top: 4px solid #0B1D9D;
border-left: 4px solid #0B1D9D;
border-bottom: 4px solid #0B1D9D;
border-radius: 6px;</string>
    </property>
    <property name="text">
     <string>L</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_out_mov">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Goudy Old Style</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Resurses_2/кремовый.jpg);
border-top: 4px solid #0B1D9D;
border-right: 4px solid #0B1D9D;
border-bottom: 4px solid #0B1D9D;
border-radius: 6px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Goudy Old Style</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Resurses_2/кремовый.jpg);
border-top: 4px solid #0B1D9D;
border-left: 4px solid #0B1D9D;
border-bottom: 4px solid #0B1D9D;
border-radius: 6px;</string>
    </property>
    <property name="text">
     <string>S</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_out_L">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Goudy Old Style</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Resurses_2/кремовый.jpg);
border-top: 4px solid #0B1D9D;
border-right: 4px solid #0B1D9D;
border-bottom: 4px solid #0B1D9D;
border-radius: 6px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resurces/res.qrc"/>
 </resources>
 <connections/>
</ui>
